app/lib/nomis_client/prisoner_category.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module NomisClient
+  class PrisonerCategory
+    class << self
+      def get(nomis_booking_id)
+        attributes_for(
+          get_response(nomis_booking_id),
+        )
+      end
+
+      def get_response(nomis_booking_id)
+        NomisClient::Base.get("/bookings/#{nomis_booking_id}").parsed
```

Comment:
Given how important this is, do we want to propagate a nomis failure and store the result of the `get`/post it to Sentry?

Reply:
Maybe we can call it a failure condition if we have a booking id and no category, actually.

Reply:
Since I'm guessing all prisoners have a category?

Reply:
Unfortunately, no, not all bookings (prisoners) seem to have a category - so we need to handle this gracefully rather than throw an error which could prevent a profile from being saved and a move from taking place. This is the output from the category synchroniser (for staging)  and you can see some have no category:

```
...
processed 975 of 995: fa8c4069-4a06-4cb3-82bf-142d39b1d827 -> Unsentenced
processed 976 of 995: fb567d4d-538b-480d-b6b7-ba520a5fe9ca -> Cat C
processed 977 of 995: fb7349a8-6362-470d-a2f3-35093a388edb -> Unsentenced
processed 978 of 995: fbb11e79-623e-4bd5-8fc7-2b0236f7a4e0 -> Cat C
processed 979 of 995: fbf9e523-6cce-4c75-aa8a-ecd687156c04 -> Cat C
processed 980 of 995: fc409b2e-0194-4d0c-9d76-5778f82f01d8 -> Cat C
processed 981 of 995: fc8fbf74-f24d-4eff-8ee7-d7054153e079 -> Cat B
processed 982 of 995: fc95c683-819b-4d07-afc1-ead49b744982 -> 
processed 983 of 995: fd2f1e07-14e2-491a-9c8f-d21f90821214 -> Unsentenced
processed 984 of 995: fd695b86-4bd6-40c6-9a8a-fe93126d3389 -> Unsentenced
processed 985 of 995: fda552e2-1139-498d-be1d-34c5c645f8e2 -> Cat C
processed 986 of 995: fe4a9556-d064-497d-80fb-81414bf3399c -> Cat C
processed 987 of 995: fe6ea530-eb8f-46bd-a553-5480f984b91f -> Unsentenced
processed 988 of 995: fec9a80c-8b2f-46bb-9469-ed83c28e60e4 -> Cat C
processed 989 of 995: ff00f677-3370-495e-b58c-95c35fc2c440 -> Cat C
processed 990 of 995: ff13213d-72a8-4654-b849-b24a7c3640a9 -> Cat C
processed 991 of 995: ff2b4570-6dce-4416-ab6b-078e4a2df7ad -> 
processed 992 of 995: ffb18270-de31-4c91-abb6-c98651582dba -> 
processed 993 of 995: fff01cea-ea4d-4a6b-b8cd-17584e4e958b -> Unsentenced
processed 994 of 995: fffed775-c645-4665-aec8-f1338bd65e83 -> Cat B
```